cadc-cdp/src/main/java/ca/nrc/cadc/cred/client/CredClient.java

```diff
@@ -255,29 +244,29 @@ public void delegate(double days)
      * @throws CertificateEncodingException
      * @throws CertificateParsingException
      * @throws CertificateExpiredException
-     * @throws CertificateNotYetValidException 
-     * @throws ca.nrc.cadc.net.ResourceNotFoundException 
+     * @throws CertificateNotYetValidException
+     * @throws ca.nrc.cadc.net.ResourceNotFoundException
      */
     public void delegate(X500Principal userDN, double days)
             throws MalformedURLException, IOException,
             InvalidKeyException, NoSuchProviderException,
             NoSuchAlgorithmException, SignatureException,
             CertificateEncodingException, CertificateParsingException,
-            CertificateExpiredException, CertificateNotYetValidException, ResourceNotFoundException
-    {
+            CertificateExpiredException, CertificateNotYetValidException, ResourceNotFoundException {
         final StringBuilder resourcePath = new StringBuilder(64);
         // user does not have the group created. Through a POST.
         // the server generates one and returns it to the user
-        if (userDN != null)
-        {
+        if (userDN != null) {
```

Comment:
Does it work if `userDN == null`?Or it gets the DN from the subject in that case?